### PR TITLE
Add help wanted label to contrib

### LINKED ISSUE #218: [FR] Update the Contributing Guide With "help wanted" Guidance
**Is your feature request related to a problem? Please describe.**
The community has asked various members of this repository "how can I help" and "where to get started".

**Describe the solution you'd like**
Add clarity to the contribution guide that identifies a label community members can search for if they need suggestions.

**Describe alternatives you've considered**
NA

**Additional context**
NA

### PR DESCRIPTION
**Pull request by peasead**
## Issues
Resolves #218 

## Summary
The community has asked various members of this repository "how can I help" and "where to get started".

This PR adds the "help wanted" label to the contributing guide.

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)? Yes
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)? Yes

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -143,7 +145,7 @@ Some of the required metadata captured in a rule file:
...
+If the relevant [categorization values](https://www.elastic.co/guide/en/ecs/current/ecs-category-field-values-reference.html) are already defined for ECS, we use these to narrow down the event type before adding the query. Typically, the query starts with the broadest grouping possible and gets narrower for each clause. For example, we might write `event.category:process and event.type:start and process.name:net.exe and process.args:group`. First, we match process events with `event.category`, then narrow to creation events with `event.type`. Of the process creation events, we're looking for the process `net.exe` with `process.name` and finally we check the arguments `group` by looking at `process.args`. This flow has little effect on the generated Elasticsearch query, but is the most intuitive to read for rule developers.
```
Comment (rw-access) [inline CONTRIBUTING.md:148]: what changed on this line? I'm losing my mind trying to decipher. It looks identical

Comment (rw-access) [reply CONTRIBUTING.md:148]: ah, guess it was a space stripped

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -67,6 +67,8 @@ Of course, feel free to bump your issues if you think they've been neglected for
 
 **Now we're talking**. If you have a bug fix or new rule that you would like to contribute to Detection Rules, please **find or open an issue about it before you start working on it.** Talk about what you would like to do. It may be that somebody is already working on it, or that there are particular issues that you should know about before implementing the change.
 
+We get asked from time-to-time if there are any rules that the community can help with, absolutely! Check out the rules with the ["help wanted" label](https://github.com/elastic/detection-rules/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22+). Don't feel like these are your only options, any Issue is up for grabs by the community, but these are rules that are good ideas and we would love to have some additional hands on.
```
Comment (rw-access) [inline CONTRIBUTING.md:70]: thanks! I think we can lowercase "Issue" -> "issue."

Comment (peasead) [reply CONTRIBUTING.md]: Fix't

**Pull request merged**